### PR TITLE
Remove references to tools.taskcluster.net in docs and hooks service test

### PR DESCRIPTION
**Pull request by arku**
Bugzilla Bug: [1562752](https://bugzilla.mozilla.org/show_bug.cgi?id=1562752)

I am not sure about the changes made to the tutorial docs, "Hello World" and "Authentication", specifically.

/cc @djmitche

### CODE REVIEW

File: ui/docs/tutorial/debug-task.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ features, debug with Taskcluster.
 You can run a task on any computer with Docker installed.
 
 First, navigate to your task in the [task
-inspector](https://tools.taskcluster.net/groups) and select the "task
+inspector](https://tc.example.com/groups) and select the "task
```
Comment (djmitche) [inline ui/docs/tutorial/debug-task.md]: Let's just remove all of the tools links in the tutorial and leave the text un-linked.  As a group, we kind of hate the tutorial and want to rewrite it from scratch, so putting much work into it probably isn't worthwhile.

File: ui/docs/tutorial/download-task-artifacts.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ let runTask = async () => {
     
   console.log("Created task:\n" + JSON.stringify(result.status, null, 2));    
   console.log("Inspect it at:");    
-  console.log("https://tools.taskcluster.net/task-inspector/#" + taskId);    
+  console.log("https://tc.example.com/tasks/" + taskId);
```
Comment (djmitche) [inline ui/docs/tutorial/download-task-artifacts.md]: This is fine though (most of these JS code snippets no longer work, to be honest)

File: ui/docs/tutorial/hello-world.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ metadata:
   name: Example Task
   description: Markdown description of **what** this task does
   owner: name@example.com
-  source: 'https://tools.taskcluster.net/task-creator/'
+  source: 'https://github.com/username/repo'
```
Comment (djmitche) [inline ui/docs/tutorial/hello-world.md]: :100:

File: ui/docs/tutorial/debug-task.md (hunk 1 above)
Comment (arku) [reply ui/docs/tutorial/debug-task.md]: Gotcha.

**Comment by arku** [comment]
@djmitche Thanks for the review. I have addressed your comments. Can you PTAL?

**Pull request merged**